File: example/test_source.py
# python test_source_instrumented.py
import logging

class Main:
    def __init__(self):
        self.a = -10
        self.b = -20

    def something_else(self, args):
        if self.a >= self.b:
          if self.a < 0:
            print "Fizz Buzz!"

        if self.a >= self.b:
          if self.a < 0:
            print "Fizz Buzz!"

    def main(self, args):
        if self.a >= self.b:
          if self.a < 0:
            print "Hello World!"

        if self.a >= self.b:
          if self.a < 0:
            print "Hello World!"

Main().main({})

File: example/test_source.java
// javac test_source_instrumented.java
// java -cp .. example.Main
package example;

import java.util.logging.Level;
import java.util.logging.ConsoleHandler;

class Main {
  public static int a = -10;
  public static int b = -20;

  public static void something_else(String[] args) {
    if (a >= b) {
      if (a < 0)
        System.out.println("Fizz Buzz!");
    }

    if (a >= b) {
      if (a < 0)
        System.out.println("Fizz Buzz!");
    }
  }

  public static void main(String[] args) {
    if (a >= b) {
      if (a < 0)
        System.out.println("Hello World!");
    }

    if (a >= b) {
      if (a < 0)
        System.out.println("Hello World!");
    }
  }
}
